lib/vanagon/driver.rb

```diff
@@ -56,26 +56,22 @@ def filter_out_components(only_build)
     end
 
     def pick_engine(options) # rubocop:disable Metrics/PerceivedComplexity
```

Comment:
Hopefully this appeases rubocop and we dont have to disable it... i think generally having a ton of return statements in a ruby method will rasie some eyebrowns.

Reply:
I tried without and rubocop complained. 

Typically, I like guardian clauses over if/elses but I agree my previous implementation was too confusing.

Reply:
ah, i had that disabled locally in a `.rubocop.yml`. sorry.